commands/action.go

```diff
@@ -25,7 +25,7 @@ import (
     "strings"
 
     "github.com/openwhisk/openwhisk-client-go/whisk"
-    "github.com/openwhisk/openwhisk-cli/wski18n"
+    "../wski18n"
```

Comment:
I doubt if you change it back to ../wski18n, go build command is able to work.
I tried with go build, it turned out ../wski18n unable to find.